OracleDatabase/README.md

```diff
@@ -135,6 +135,9 @@ Another option is to use `docker exec` and run `sqlplus` from within the same co
 
 	docker exec -ti <container name> sqlplus pdbadmin@ORCLPDB1
 
+## Known issues
+* OverlayFS (`overlay`) on CentOS has proven to run into Docker bug #25409. We recommend using `btrfs` or `overlay2` instead. For more details see issue #317.
```

Comment:
It's not OverlayFS causing the problem, it's the `overlay` graph driver for Docker. Both the `overlay` and `overlay2` graph drivers use the same underlying OverlayFS in the kernel.